src/Model/ModelUtilities/FlowModelInterface.f90

```diff
@@ -435,10 +435,10 @@ subroutine initialize_bfr(this)
   end subroutine initialize_bfr
 
   !> @brief Advance the budget file reader
-  !!
-  !! Advance the budget file reader by reading the next chunk
-  !! of information for the current time step and stress period.
-  !!
+  !
+  ! Advance the budget file reader by reading the next chunk
+  ! of information for the current time step and stress period.
+  !
```

Comment:
From what I can tell, the majority (if not all) of the MF6 code base uses "!!" for lengthier docstrings as opposed to a single "!".  I think it would be best to stay consistent in this.

Reply:
think this was caused by running fprettify without the config file arg &mdash; fixed